src/components/MetricsPane/charts/AggregateFishBiomas.jsx

```diff
@@ -26,14 +26,28 @@ export const AggregateFishBiomass = () => {
 
   const plotlyDataConfiguration = [
     {
-      x: surveyFishbeltBiomassValues.map((value) => (value > 5000 ? 5000 : value)),
+      x: surveyFishbeltBiomassValues.filter((value) => value <= 5000),
       type: 'histogram',
+      name: '',
```

Comment:
Just curious what an empty name property does for Plotly?

Reply:
Hide the trace name when hovering